tests/HasMetaTest.php

```diff
@@ -9,6 +9,17 @@ public function testUserHasMeta()
         $post = Post::published()->hasMeta('username', 'juniorgrossi')
             ->first();
 
+        $posts = Post::published()->hasMeta('city', ['amsterdam', 'london'])
```

Comment:
Update the `database.sql` file in `tests/config` with your local sql dump, to make this test passing.